### DISCUSSION TITLE
probleme de fonctionnement de certains champs filament dans des composant livewire

### DISCUSSION THREAD

**Discussion by franckopheno** [filamentphp/filament#13846] [Help]
### Package

Form builder

### Package Version

v3.2

### How can we help you?

j'essaie d'utiliser certain champs specifique de filament (taginput,richeditor,markdownEditor) dans mes composant livewire .mais j'ai toujour une erreur lors de la soumission vers le controleur livewire comme quoi le champs ne serait pas renseigner ....pourtant j'ai remplie les donnees du champ cote front ..

**Comment by leandrocfe** [comment]
Write questions in English please
